frontend/kesaseteli/employer/src/components/application/form/TextInput.tsx

```diff
@@ -16,6 +16,7 @@ export type TextInputProps = {
   placeholder?: string;
   helperFormat?: string;
   onChange?: (value: string) => void;
+  autoComplete?: string;
```

Comment:
wouldn't it be better set the actual available values there? `"on"|"off"|""`

Reply:
Oh, there are actually quite a lot of those alternatives as shown here https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete. Could the type be linked from React or HDS or somewhere?

Reply:
I checked from the React `index.d.ts` and it seems to be a `string` there as well!

- [InputHTMLAttributes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L2364)
- [FormHTMLAttributes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L2282)